File: data/java/COUNT_NUMBER_WAYS_REACH_GIVEN_SCORE_GAME_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(83);
    param0.add(29);
    param0.add(17);
    param0.add(12);
    param0.add(93);
    param0.add(55);
    param0.add(97);
    param0.add(75);
    param0.add(22);
    param0.add(52);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("COUNT_NUMBER_WAYS_REACH_GIVEN_SCORE_GAME," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("COUNT_NUMBER_WAYS_REACH_GIVEN_SCORE_GAME," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}

File: data/python/COUNT_NUMBER_WAYS_REACH_GIVEN_SCORE_GAME_MAIN.py
if __name__ == '__main__':
    param = [
    (83,),
    (29,),
    (17,),
    (12,),
    (93,),
    (55,),
    (97,),
    (75,),
    (22,),
    (52,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("COUNT_NUMBER_WAYS_REACH_GIVEN_SCORE_GAME," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("COUNT_NUMBER_WAYS_REACH_GIVEN_SCORE_GAME," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
